acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/CmdLineParamsDefaultImpl.java

```diff
@@ -212,7 +224,48 @@ private Collection<String> createEth2Args() {
     return params;
   }
 
-  private String createCommaSeparatedList(final List<String> values) {
+  private String createCommaSeparatedList(final Collection<String> values) {
```

Comment:
Name is a little bit misleading as it creates a string, maybe createCommaSeperatedString?

Reply:
yes, agreed.

Reply:
Sounds more like a `join` operation to me

Reply:
`joinWithCommas`?

Reply:
I've removed the method and instead use `String.join(",",...);` directly since its quite obvious.